beacon-chain/verification/fake.go

```diff
@@ -3,21 +3,7 @@ package verification
 import (
 	"testing"
 
-	fieldparams "github.com/OffchainLabs/prysm/v6/config/fieldparams"
-
 	"github.com/OffchainLabs/prysm/v6/consensus-types/blocks"
-	"github.com/OffchainLabs/prysm/v6/consensus-types/primitives"
-)
-
-type (
```

Comment:
This part should have been moved to `testing/util/` in the previous PR but I forgot to do it (Preston's comment).
Doing it now.

Note: `testing/util` has a `testonly = True,` parameter in its `BUILD.bazel`, contrary to this `fake` package.